connector/MongoDBAtlasODBC.pq

```diff
@@ -1,12 +1,11 @@
-[Version = "0.0.0"]
 section MongoDBAtlasODBC;
 
 // When set to true, additional trace information will be written out to the User log.
 // This should be set to false before release. Tracing is done through a call to
 // Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a
 // no-op and simply returns the original value.
 EnableTraceOutput = false;
-
+Version = "0.0.0";

```

Comment:
I moved this to a property that was more accessible.

Reply:
I don't know if I want to go through yet another back and forth with MSFT.
Can you leave it as it was before, so that it follows [their guidelines](https://learn.microsoft.com/en-us/power-query/handling-versioning)?

Reply:
Thanks @nbagnard for pointing out the Version in the guidelines.

The original `[Version = "0.0.0"]` seems to not be accessible within the section where APPNAME is set.  I am okay with defining it twice and modifying our `sed` command to update both values.

Reply:
I've reverted this and gone with a different approach that takes away the need for the Text.Format